platform/darwin/src/MGLShapeSource.h

```diff
@@ -321,6 +323,49 @@ MGL_EXPORT
  */
 - (NSArray<id <MGLFeature>> *)featuresMatchingPredicate:(nullable NSPredicate *)predicate;
 
+/**
+ Returns an array of map features that are the leaves of the specified cluster.
+ ("Leaves" are the original points that belong to the cluster.)
+ 
+ This method supports pagination; you supply an offset (number of features to skip)
+ and a maximum number of features to return.
+ 
+ @param cluster An object that conforms to the `MGLCluster` protocol. Currently
+    the only types that can conform are private subclasses of `MGLPointFeature`.
+ @param offset Number of features to skip.
+ @param limit Maximum number of features to return
+ 
+ @return An array of objects that conform to the `MGLFeature` protocol.
+ */
+- (NSArray<id <MGLFeature>> *)leavesOfCluster:(id<MGLCluster>)cluster offset:(NSUInteger)offset limit:(NSUInteger)limit;
```

Comment:
MapKit’s API refers to a cluster’s constitutent features as “members”. I think “member” is more self-explanatory than “leaf”; the analogy to trees requires some explanation.